### PR TITLE
feat(farm-v1): depositAndJoin

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: .env.adpr
Comment (namgold) [inline .env.adpr:1]: revert when release

File: src/state/farms/elastic/updaters/v1.tsx [hunk 1]
```diff
@@ -163,7 +163,7 @@ const FarmUpdaterV1: React.FC<CommonProps> = ({ interval }) => {
     const i = interval
       ? setInterval(() => {
           getElasticFarms()
-        }, 10_000)
+        }, 10_0000)
```
Comment (namgold) [inline src/state/farms/elastic/updaters/v1.tsx:166]: :pepe-hanged:

Comment (namgold) [reply src/state/farms/elastic/updaters/v1.tsx:166]:
```suggestion
        }, 100_000)
```

File: .env.adpr
Comment (viet-nv) [reply .env.adpr:1]: :hihi:

File: src/state/farms/elastic/updaters/v1.tsx (hunk 1 above)
Comment (viet-nv) [reply src/state/farms/elastic/updaters/v1.tsx]: :hihi:

**Review by namgold — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**